android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -163,6 +163,8 @@ private int getIconResourceId(Bundle bundle){
 
     public void sendToNotificationCentre(Bundle bundle) {
         try {
+            Class intentClass = getMainActivityClass();
```

Comment:
was the framework always using this class? Was there a behaviour like "if I click on a group notification -> it takes you to that group screen"?

If there was, we might be breaking that.